setup.py

```diff
@@ -45,6 +45,7 @@ def find_requires():
     zip_safe=False,
     install_requires=find_requires(),
     include_package_data=True,
+    setup_requires=['sphinx'],
```

Comment:
What's the reason for always requiring sphinx ? To me it should be optional dependency.